### ISSUE TITLE
Week 25: Audio Analysis Integration

### ISSUE THREAD

**Issue by Tlewis10** [gw-sd-2016/Presenting101#11]
@twood02 This week I tried combining my audio analysis components. However, it seems like the libraries conflict with each other so I spent the majority of the time trying to figure out this and then understand why it was still not working. I will continue to try to get everything together, if I continue to have problems integrating them I will have to run them as separate programs (it still works are needed though). Next week I will continue to do this, create the user error report, and make improvements to the visual analysis and UI.

**Comment by twood02** [comment]
have you verified that they work when run as separate programs so that you
know that is a safe fall back?

**Comment by Tlewis10** [comment]
Yes I tested it to make sure there wasn't something wrong with how I was
doing it.

Thanks,

Terrence D. Lewis
The George Washington University
School of Engineering and Applied Science
Bachelor of Science, Computer Science; Expected 2016
tlewis@gwmail.gwu.edu
Please excuse any errors or typos. Sent from my LG G Stylo.

**Comment by twood02** [comment]
OK, that's good at least.  If you want to sign up for a meeting slot you
can do so here: https://calendly.com/timwood/meetings/03-03-2016
